starknet_py/net/client_models.py

```diff
@@ -325,7 +325,7 @@ class EntryPointsByType:
 
 
 @dataclass
-class DeclaredContract:
+class ContractClass:
     """
     Dataclass representing contract declared to starknet
     """
```

Comment:
What with the program field? Don't we have to specify this? Would it be useful?

Reply:
There are no use cases for typed program currently. Maybe in the future, after 0.11.0.

Reply:
Ok 👌